buildlib/pr/main.yml

```diff
@@ -173,16 +177,25 @@ stages:
               CONTAINER: ubuntu2004
               long_test: yes
               test_static: yes
+              extra_modules: ""
+              extra_tls: ""
             ubuntu1804:
               CONTAINER: ubuntu1804
               test_static: yes
+              extra_modules: ""
+              extra_tls: ""
             sles15sp2:
               CONTAINER: sles15sp2
             rhel82:
               CONTAINER: rhel82
             fedora34:
               CONTAINER: fedora34
               long_test: yes
+            centos7:
+              CONTAINER: centos7_ib
+              test_static: yes
+              extra_modules: ucx-ib
+              extra_tls: ib
```

Comment:
shouldn't this be "rc_verbs rc_mlx5 ..."

Reply:
do you mean check for transport? (now it checks for component)

Reply:
ok, script will look for transport